movies_app/templates/movies_app/review_update.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'movies_app/base.html' %}
+{% load static %}
+
+{% block title %}Edit Review{% endblock %}
+
+{% block head %}
+<link rel="stylesheet" type="text/css" href="{% static 'css/styles.css' %}">
+{% endblock %}
```

Comment:
no i w ogóle nie macie takiego bloku `head` w `base.html`, więc to nie działa. Do wyrzucenia, bo i tak nie powinno tego być